core/txpool/legacypool/legacypool.go

```diff
@@ -748,7 +748,7 @@ func (pool *LegacyPool) add(tx *types.Transaction, local bool) (replaced bool, e
 		// If the new transaction is underpriced, don't accept it
 		// if !isLocal && pool.priced.Underpriced(tx) {
 		if !isLocal && tx.GasFeeCapIntCmp(pool.gasTip.Load()) < 0 {
-			log.Trace("Discarding underpriced transaction", "hash", hash, "gasTipCap", tx.GasTipCap(), "gasFeeCap", tx.GasFeeCap())
+			log.Trace("Discarding underpriced transaction", "hash", hash, "gasTipCap", tx.GasTipCap(), "gasFeeCap", tx.GasFeeCap(), "txPoolGasTip", pool.gasTip.Load())
```

Comment:
sorry that I found a potential data inconsistency that is introduced by my comment... if `pool.gasTip.Store()` is called in between (after the GasFeeCapIntCmp check and before printing the log), `pool.gasTip.Load()` would be different.

and `pool.gasTip.Store()` is only called by `SetGasTip` after the legacy pool is initialized. What do you think, @HAOYUatHZ? I'm OK with any refactors about this.

Reply:
I think we can keep it as what it is now